baremetal/metal3machine_manager.go

```diff
@@ -1326,7 +1328,7 @@ func (m *MachineManager) GetProviderIDAndBMHID() (string, *string) {
 		m.Log.Info("ProviderID is in new format, it does not contain the BMH ID", "providerID", *providerID)
 		return *providerID, nil
 	}
-	m.Log.Info("ProviderID contains the BMH ID", "providerID", *providerID)
+	m.DebugLog.Info("ProviderID contains the BMH ID", "providerID", *providerID)
```

Comment:
I think it would be fine to just do `m.Log.V(4).Info("...")`.
After checking that [CAPI logging guidelines](https://main.cluster-api.sigs.k8s.io/developer/logging.html#log-levels), I would say maybe level 3 or 4 makes sense here.

Reply:
updated, thanks!